website/docs/r/organization_token.html.markdown

```diff
@@ -28,6 +28,28 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) The token's expiration date. The expiration date must be a date/time string in RFC3339 
+format (e.g., "2024-12-31T23:59:59Z"). If no expiration date is supplied, the expiration date will default to null and 
+never expire.
+
+## Example Usage
+
+When a token has an expiry:
+
+```hcl
+resource "tfe_organization_token" "test" {
+  organization = "my-org-name"
+}
+
+resource "time_rotating" "example" {
+  rotation_days = 30
+}
+
+resource "tfe_organization_token" "test" {
+  organization = data.tfe_organization.org.name
+  expired_at = time_rotating.example.id
```

Comment:
OK so I tested this out and the docs for the provider seem to be wrong. `id` returns the base timstamp and `rotation_rfc3339` returns the offset timestamp. The examples should look like this:

```suggestion
  expired_at = time_rotating.example.rotation_rfc3339
```

Reply:
Fix here https://github.com/hashicorp/terraform-provider-time/pull/137

Reply:
Good to know, thank you! I've updated the examples